artemis-server/src/test/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImplTest.java

```diff
@@ -962,6 +963,7 @@ private void checkScaleDownConfiguration(ScaleDownConfiguration scaleDownConfigu
       assertEquals("g0", scaleDownConfiguration.getGroupName());
       assertEquals("c0", scaleDownConfiguration.getClusterName());
       assertFalse(scaleDownConfiguration.isEnabled());
+      assertEquals(33, scaleDownConfiguration.getCommitInterval());
```

Comment:
a very tiny request.. can you use a different constant here? you used 33 on the XML... if the XML is parsed by accident in the test you wouldn't know if the value came from the XML or the properties.


If you okay with merging this now I can make this change myself with a git checkout and push.